### ISSUE TITLE
Parameter cacerts does not accept filename

### ISSUE THREAD

**Issue by rethab** [compose/transporter#332]
## Bug report
The sample config for mongodb suggests that there is a parameter `cacerts` that accepts a list of filenames. However from what I can tell, this parameter actually accepts a (list of-) certificates rather than their name.

I would like to think I'm using the whole thing incorrectly, because this seems so trivial.. However I cannot see another way to get this working by passing a filename. Do I really have to pass in the certificate there? That doesn't sound very intuitive to me.

### System info:

- 0.3
- Linux
- MongoDB 3.2

### Reproducible Steps:

1. Put path of cert in `cacerts`  --> Doesn't work
2. `clip -sel clip < path_of_ca_cert`, put this into `cacerts` --> Works

### Proposal:
- Accept filename in `cacerts`

**Comment by jipperinbham** [comment]
@rethab thanks for the detailed report

it should definitely be accepting a path to a file and not the raw cert string, we clearly introduced a regression somewhere. 

I'll take a look at getting this fixed soon but it appears as though you do have a workaround for now?

**Comment by rethab** [comment]
Hi @jipperinbham, 

I'm just about to get started with this and I guess the workaround is okay for now.

Thanks for taking a look into this!

- Reto

**Comment by rethab** [comment]
Hi @jipperinbham, I'm afraid the workaround doesn't really work either. So what I tried was to read the certificate into an environment variable and use that for the `cacerts` parameter. However since my certificate is formatted with newlines, I got an error:
```
SyntaxError: (anonymous): Line 4:15 Unexpected token ILLEGAL (and 8 more errors)
```
I only got it to work by editing the pem file, putting everything on one line and using `\n` as separator. However editing the pem file is not really a viable option for me.

- Reto
